addons/website/static/src/scss/website.backend.scss

```
@@ -99,7 +99,8 @@
     box-shadow: 0 4px 20px rgba(0, 0, 0, 0.15),
                 16px 12px 32px rgba(0, 0, 0, 0.15),
                 64px 0 64px rgba(0, 0, 0, 0.15);
-    border-radius: 2rem;
+    border-radius: 2.5rem;
```

While this may work for most desktop viewports, issues arise on screens with significantly greater height (e.g. portrait monitors).

The problem is that rem is a fixed unit based on the root font size. It doesn't scale visually when the container grows (e.g. due to height rule set at 92% (L93) on larger screens, or the scaling logic triggered below 800px). This causes the border-radius to appear inconsistent.

I think it would be better to use vh , which would scales the border-radius proportionally.

```suggestion
    border-radius: 3.5vh;
```